tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
For the cdk project we extend the guardian tsconfig - we might want to do that here? it looks like this:

```
{
  "extends": "@guardian/tsconfig/tsconfig.json",
  "compilerOptions": {
    "module": "CommonJS"
  },
}

```